feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/handlers/feedingpoint/FeedingPointHandler.kt

```diff
@@ -8,7 +8,7 @@ internal interface FeedingPointHandler {
 
     suspend fun fetchFeedingPoints()
 
-    fun hideOtherFeedingPoints(feedingPoint: FeedingPointModel)
```

Comment:
renamed this function as it got additional functionality of applying visual "reserved" look for feeding point